web3/providers/eth_tester/middleware.py

```diff
@@ -35,7 +36,11 @@ def is_named_block(value):
     return value in {"latest", "earliest", "pending"}
 
 
-to_integer_if_hex = apply_formatter_if(is_string, hex_to_integer)
+def is_str_is_hex(value):
```

Comment:
Would naming this `is_hexstr` be more consistent with our existing naming convention?